README.md

```diff
@@ -39,6 +39,7 @@ For more information about communication, refer to the [Ansible Communication gu
 
 - **Modules**:
   - [proxysql_backend_servers](https://docs.ansible.com/ansible/latest/modules/proxysql_backend_servers_module.html)
+  - [proxysql_info]((https://docs.ansible.com/ansible/latest/modules/proxysql_info_module.html))
   - [proxysql_global_variables](https://docs.ansible.com/ansible/latest/modules/proxysql_global_variables_module.html)
```

Comment:
```suggestion
  - [proxysql_global_variables](https://docs.ansible.com/ansible/latest/modules/proxysql_global_variables_module.html)
  - [proxysql_info](https://docs.ansible.com/ansible/latest/modules/proxysql_info_module.html)
```
I thought of this approach. Maybe we can just change it to `community.dns`'s format https://github.com/ansible-collections/community.dns, i.e. module name + short description. Because the doc is generated periodically and there can be confusion with non existent doc pages later (we can change it later in a separate PR if you're OK with the suggestion).

Reply:
I'm fine with it. And yeah, lets do it in a separate PR.